# Define the goal
geometry_msgs/PoseStamped target_pose
---
# Define the result
uint32 total_dishes_cleaned
---
# Define a feedback message
geometry_msgs/PoseStamped base_position
